openedx/core/lib/api/utils.py

```diff
@@ -0,0 +1,57 @@
+import functools
+from importlib import import_module
+
+from django.conf import settings
+
+
+def pluggable_override(override):
+    """
+    This decorator allows overriding any function or method by pointing to an alternative implementation
+    with `override` param.
+    :param override: path to the alternative function
+
```

Comment:
We should start out right, and give explicit instructions about how best to document uses of this decorator.  These will be introducing new settings that need explanataion.

Reply:
Could you please give some more details about what you meant here?

Reply:
@nedbat, I moved this change to edx/edx-django-utils#64, as @nasthagiri suggested. We can continue this conversation there.